src/raglite/_embed.py

```diff
@@ -201,3 +191,20 @@ def embed_sentences(sentences: list[str], *, config: RAGLiteConfig | None = None
     else:
         sentence_embeddings = _embed_sentences_with_windowing(sentences, config=config)
     return sentence_embeddings
+
+
+def embed_text(text: list[str], *, config: RAGLiteConfig | None = None) -> FloatMatrix:
```

Comment:
I was thinking about how this function signature, and have some thoughts to (hopefully) improve the API:
1. `text` seems to imply natural language, whereas this function can embed any list of strings (such as code or data for example).
2. We already have an (admittedly private) `_embed_string_batch(string_batch: list[str])` function signature already.

Given the above I think this would be a small improvement:
```suggestion
def embed_strings(strings: list[str], *, config: RAGLiteConfig | None = None) -> FloatMatrix:
```